### ISSUE TITLE
Scroll View - scrollbar not showing in iOS 14 devices

### ISSUE THREAD

**Issue by paola-guitian-papa** [facebook/react-native#30067]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
Using the Scroll View component does not show the scrollbar on iOS 14 devices. The scrollbar does appear on andorids and iOS 13 devices but not on devices with iOS 14.
 
## React Native version:
0.62.2

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Import and use a Scroll View. Make sure there is enough content to activate scrolling.
2. Test on iOS 13 device and test on iOS 14 device. You will notice that the scrollbar appears on iOS 13 but not on 14.

## Expected Results
Scrollbar appears on iOS14

**Comment by keremoge** [comment]
Can you please try to **turn off** `Reduce Motion` and `Prefer Cross-Fade Transitions` one by one?
If the problem goes away than still there is a problem with **iOS14 motions + react-native** :)
<img width="278" alt="Screen Shot 2020-10-03 at 10 41 59" src="https://user-images.githubusercontent.com/4692074/94986264-085c4580-0566-11eb-8c84-41e4acba519e.png">

**Comment by HiveHicks** [comment]
@keremoge hey! I'm investigating a bug with scrolling that some of our users are experiencing on iOS 14. We're not using React Native, but your comment seems interesting to me, as I also thought it might have something to do with accessibility settings. Is there anywhere I can read more about how Reduce Motion affects UIScrollView behavior?

**Comment by czekFREE** [comment]
I am probably facing same issue - no scrollbars in ScrollView/FlatList components. 

iOos version: 14.0.1
RN version: 0.63.3

**Comment by keremoge** [comment]
> @keremoge hey! I'm investigating a bug with scrolling that some of our users are experiencing on iOS 14. We're not using React Native, but your comment seems interesting to me, as I also thought it might have something to do with accessibility settings. Is there anywhere I can read more about how Reduce Motion affects UIScrollView behavior?

Actually, it shouldn't effect in a bad way. We learnt the bug  by chance, one of our cutomer found the temp fix :) then we tried and see that react native + ios14/ios14.0.1 have a side affect on scrollbars when you reduce motion and change cross-fade transitions. I'm still waiting a RN fix. If there will not be a fix then we will try to manually fix it. I think it is a problem RN

**Comment by Whichoney** [comment]
Did anybody make any progress on this issue &mdash; as we're also experiencing the same?

**Comment by Whichoney** [comment]
Just a thought guys: we were having the same issue, but it's because the user had dark mode enabled, whereas our app doesn't _yet_ support dark mode (still has a white background for example). By default the scrollbar will be white and so not visible, unless you set `indicatorStyle="black"` for your `<ScrollView />`.

**Comment by neiker** [comment]
Setting the `indicatorStyle` works for me, thanks @Whichoney 

I set it as default prop so it affects all the scrolleables:

On the `App.tsx`:
```ts
ScrollView.defaultProps = {
  indicatorStyle: "black"
};
```
